### ISSUE TITLE
What do obs and preds mean?

### ISSUE THREAD

**Issue by rlarudgus99** [FGiuliari/Trajectory-Transformer#27]
Hi! Thanks for your great work!

I am studying your research.
I just want to know the meaning of the obs and preds that baselineUtils.py uses to slice an array, respectively.

Also, among the get_strid_data_clust functions
```
ped = raw_data.ped.unique()
frame=[]
ped_ids=[]
for p in ped:
    for i in range(1+(raw_data[raw_data.ped == p].shape[0] - gt_size - horizon) // step):
        frame.append(dt[dt.ped == p].iloc[i * step:i * step + gt_size + horizon, [0]].values.squeeze())
        inp_te.append(raw_data[raw_data.ped == p].iloc[i * step:i * step + gt_size + horizon, 2:4].values)
        ped_ids.append(p)

frames=np.stack(frame)
inp_te_np = np.stack(inp_te)
ped_ids=np.stack(ped_ids)
```
What does fram mean in the part?

I'd like to ask you something else.
The following formula is used to calculate loss

`loss = F.pairwise_distance(pred[:, :,0:2].contiguous().view(-1, 2), ((batch['trg'][:, :, 2:4].to(device)-mean.to(device))/std.to(device)).contiguous().view(-1, 2).to(device)).mean() + torch.mean(torch.abs(pred[:,:,2]))`

Why add the absolute value mean of the Output Feature, pred[:, :, 2] , which is not used for prediction??

Looking forward to your reply!
